### PR TITLE
308/insert images

### PR DESCRIPTION
**Pull request by hirunatan**

### CODE REVIEW

File: common/uxbox/common/geom/shapes.cljc [hunk 1]
```diff
@@ -191,6 +191,17 @@
...
+(defn resize
+  [shape width height]
+  (us/assert map? shape)
+  (us/assert number? width)
+  (us/assert number? height)
+  (assoc shape
+         :width width
+         :height height
+         :x2 (+ (:x1 shape) width)
+         :y2 (+ (:y1 shape) height)))
```
Comment (Alotor) [inline common/uxbox/common/geom/shapes.cljc]: You need to update the selrect `(update :selrect resize width height)`

File: frontend/src/uxbox/main/ui/workspace/drawarea.cljs [hunk 2]
```diff
@@ -74,6 +74,66 @@
...
+  (let [tool (seek #(= type (:type %)) minimal-shapes)]
+    (assert tool "unexpected drawing tool")
+    (assoc tool
+           :id (uuid/next)
+           :x 0
+           :x1 0
+           :x2 0
+           :y 0
+           :y1 0
+           :y2 0
```
Comment (Alotor) [inline frontend/src/uxbox/main/ui/workspace/drawarea.cljs]: You don't need to add x1, x2, y1 and y2. These are fields only in selrect. Also, this parameters can break the application. It doesn't make sense a 0 width or height. The minimum should be 1

File: common/uxbox/common/geom/shapes.cljc (hunk 1 above)
Comment (hirunatan) [reply common/uxbox/common/geom/shapes.cljc]: fixed

File: frontend/src/uxbox/main/ui/workspace/drawarea.cljs (hunk 2 above)
Comment (hirunatan) [reply frontend/src/uxbox/main/ui/workspace/drawarea.cljs]: fixed

**Review by Alotor — APPROVED** [review]

**Pull request merged**